fundraiser/project/fundraiser-contract/tests/functions/unpledge.rs

```diff
@@ -458,10 +458,10 @@ mod revert {
     }
 
     #[tokio::test]
-    #[should_panic(expected = "Revert(18446744073709486080)")]
+    #[should_panic(expected = "AlreadyClaimed")]
     async fn after_claimed() {
         let (author, user, asset, _, defaults) = setup().await;
-        let deadline = 6;
+        let deadline = 7;
```

Comment:
Not sure where this hardcoded value comes from but it may be useful to use `provider.produce_blocks(duration).await;` and `provider.latest_block_height().await.unwrap()`

Reply:
oh right, I had that on my mind but I forgot to update it. Will do that soon

Reply:
I knew I've done it somewhere. The changes for that can be seen in #322 . Once this bump goes in then the provider will be used throughout when that PR goes in